boardgame-ui/src/components/timer/AddTimerForm.vue

```diff
@@ -0,0 +1,298 @@
+<template>
+  <div class="wrapper">
+    <b-loading :active="isLoading"/>
+    <form @submit.prevent="createTimer('form-timerCreation')" data-vv-scope="form-timerCreation" v-if="timer">
+      <h1 class="title">{{$t('timer.add-edit.timer.title')}}</h1>
```

Comment:
Display different label depending on whether it is creation or edition?

Reply:
The edition will most likely not be done through this form, as it will be done through the websocket and therefore allow a more fluent interaction. That is why I have fixed the title to create.